postcss.config.js

```diff
@@ -0,0 +1,3 @@
+const config = require('./packages/terra-site/postcss.config');
+
+module.exports = config;
```

Comment:
this file doesn't do anything right now, but it will be used by storybook once its added and will ensure storybook uses the same postcss config as terra-site.